article/related-work.tex

```diff
@@ -10,13 +10,48 @@ \subsection{Light GCN}
-This is done with the purpose of calculating how much the item contributes to the users preference \cite{NGCF_2019}. % WTF is this shit????
+This is done with the purpose of calculating how much the item contributes to the users preference \cite{NGCF_2019}.
 LightGCN criticize the use of the weight matrices $W_1$ and $W_2$ as not being useful for CF as each user-item interaction graph only has the ID as input and it has no semantic value \cite{lightgcn}.
 Also within the second component the message aggregation is implemented as:
 \begin{equation}
     e_{u}^{(l)} = \mbox{LeakyReLU}(m^{(l)}_{u \leftarrow u} + \sum^{}_{i \in \mathcal{N}} m^{(l)}_{u \leftarrow i}),
 \end{equation}
 where $e_{u}^{(l)}$ are the embeddings of user $u$ at layer $l$.
-LeakyReLU is the activation function chosen in NGCF to allow encoding positive and small negative signals. % WHAT DOES THIS EVEN MEAN???
+LeakyReLU is the activation function chosen in NGCF to allow encoding positive and small negative signals.
 LightGCN shows that NGCF will perform better if the feature transformation is removed, and that the activation function has small effect when the feature transformation is included, but if feature transformation is disabled the activation function will have a negative impact on performance.
 Light GCN also shows that NGCF will significantly improve if both the activation function and feature transformation are removed \cite{lightgcn}.
+
+\subsection{Price-aware Recommendation with Graph Convolutional Networks}
+Yu Zheng et al. develops an effective method called \textit{Price-aware User Preference-modeling (PUP)} that is used to create recommendations with focus on the price factor \cite{Priceaware}.
+There are two main challenges where the first is that the preferences of a user on item price are unknown, and can only be seen implicitly by previous purchases.
+The second challenge is that the price of an item largely depends on what category the item is within.
+They solve this by integrating the categories and items into the propagation process.
+PUP represents the data as an heterogeneous undirected graph $G = (V,E)$ where the nodes in $V$ consists of user nodes $u \in U$, item nodes $i \in I$, category nodes $c \in \textbf{c}$, and price nodes $p \in \textbf{p}$.
+The edges in $E$ consists of interaction edges $(u, i)$ with $R_{ui} = 1$ if there is an interaction between user $u$ and item $i$ and category edges $(i, \textbf{c}_i)$ and price edges $(i, \textbf{p}_i)$.
+Traditional Latent Factor Models such as Matrix Factorization is with this graph insufficient for Recommendation due to being more pairs than $(u,i)$, but also $(i,p)$ and $(i,c)$ \cite{Priceaware}.
+Hereby Yu Zheng et al. use Graph Neural Network to learn the embeddings so that each node has a separate embedding $e' \in \mathbb{R}^d$ where d is the dimensions of the embedding.
```

Comment:
```suggestion
Hereby Yu Zheng et al. use a Graph Neural Network to learn the embeddings so that each node has a separate embedding $e' \in \mathbb{R}^d$ where d is the dimensions of the embeddings.
```